server/service/integration_mdm_dep_test.go

```diff
@@ -3726,3 +3726,119 @@ func (s *integrationMDMTestSuite) TestSetupExperienceFlowCancelScript() {
 	require.Len(t, cmds, 1)
 	require.Equal(t, "DeviceConfigured", cmds[0].Command.RequestType)
 }
+
+func (s *integrationMDMTestSuite) TestDeleteMultipleHostsPendingDEP() {
+	t := s.T()
+
```

Comment:
You probably want to call `s.setSkipWorkerJobs(t)` to prevent worker from running at non-deterministic times?  (or is that the default now?)